geop/src/main/scala/com/azavea/usace/programanalysis/geop/LayerReader.scala

```diff
@@ -0,0 +1,54 @@
+package com.azavea.usace.programanalysis.geop
+
+import geotrellis.raster.Tile
+import geotrellis.spark.io._
+import geotrellis.spark.{SpatialKey, TileLayerMetadata, _}
+import geotrellis.spark.io.s3.{S3AttributeStore, S3ValueReader}
+import geotrellis.vector.{Extent, MultiPolygon}
+
+import org.apache.spark.SparkContext
+
+object LayerReader {
+  /**
+    * Given values for zoom, x and y, and a spark context, returns
+    * the corresponding [[Tile]] object.
+    *
+    * @param    zoom  [[Int]] value of zoom level
+    * @param    x  [[Int]] x-value of the spatial key
+    * @param    y  [[Int]] y-value of the spatial key
+    * @param    sc              [[SparkContext]] for the [[S3LayerReader]]
```

Comment:
Should `S3LayerReader` here be `S3ValueReader` instead?

Also consider aligning these to make them more readable:

```diff
diff --git a/geop/src/main/scala/com/azavea/usace/programanalysis/geop/LayerReader.scala b/geop/src/main/scala/com/azavea/usace/programanalysis/geop/LayerReader.scala
index 5144a62..bdd3c3a 100644
--- a/geop/src/main/scala/com/azavea/usace/programanalysis/geop/LayerReader.scala
+++ b/geop/src/main/scala/com/azavea/usace/programanalysis/geop/LayerReader.scala
@@ -14,10 +14,10 @@ object LayerReader {
     * the corresponding [[Tile]] object.
     *
     * @param    zoom  [[Int]] value of zoom level
-    * @param    x  [[Int]] x-value of the spatial key
-    * @param    y  [[Int]] y-value of the spatial key
-    * @param    sc              [[SparkContext]] for the [[S3LayerReader]]
-    * @return                   [[Tile]] object
+    * @param    x     [[Int]] x-value of the spatial key
+    * @param    y     [[Int]] y-value of the spatial key
+    * @param    sc    [[SparkContext]] for the [[S3LayerReader]]
+    * @return         [[Tile]] object
     */
   def apply(
     zoom: Int,
```